sashimi-webapp/index.html

```diff
@@ -28,6 +28,13 @@
       rel="stylesheet">
   </head>
   <body>
+    <!-- js-sequence-diagram does not support markdown-it, hence we are doing this -->
+    <script src="./vendors/webfont.js"></script>
+    <script src="./vendors/snap.svg-min.js"></script>
+    <script src="./vendors/underscore-min.js"></script>
+    <script src="./vendors/sequence-diagram.js"></script>
+    <script src="./vendors/raphael.min.js"></script>
+    <script src="./vendors/flowchart-latest.js"></script>
```

Comment:
We definitely should use webpack to load these dependency next time. But let's leave that for next milestone #281